include/mbgl/util/image.hpp

```diff
@@ -26,6 +28,18 @@ class Image {
           height(h),
           data(std::move(data_)) {}
 
+    Image(Image&& o)
+        : width(o.width),
+          height(o.height),
+          data(std::move(o.data)) {}
+
+    Image& operator=(Image&& o) {
```

Comment:
>  What was the rationale here?

Like other `mbgl::utill::noncoypable` classes, I think we want to prevent copying an `Image` by design.